XenModel/XenAPI-Extensions/VM.cs

```diff
@@ -1022,17 +1022,19 @@ public enum VmTemplateType
             CoreOS = 4,
             Debian = 5,
             NeoKylin = 6,
-            Oracle = 7,
-            RedHat = 8,
-            SciLinux = 9,
-            Suse = 10,
-            Ubuntu = 11,
-            Citrix = 12,
-            Solaris = 13,
-            Misc = 14,
-            Snapshot = 15,
-            SnapshotFromVmpp = 16,
-            Count = 17  //bump this if values are added
+            Asianux  = 7,

```

Comment:
Hi,Mihaela. I had changed the icon with a grey border and also the order of linux vms. 
Please review it.  Thanks a lot.